API_flask.py

```diff
@@ -0,0 +1,85 @@
+import checkmanga
+from flask import Flask, request, jsonify
+app = Flask(__name__)
+
+checkmangautil = checkmanga.CheckMangaUtil('mycheckmanga.db')
+
+def validate_input_json_keys(request_json, required_keys):
+	for key in required_keys:
+		if key not in request_json.keys():
+			return key
+	return True
+
+def process_json_wrapper(request, func, route_name, required_keys):	
```

Comment:
you can actually make this into a decorator! it would be really pretty and clean up a loooot of code here

Reply:
I was thinking that but wasn't sure whether to put it before or after the @ app.route() Flask decorator. I'm thinking it should probably go:

@ app.route()
@ process_json_wrapper()
def func():